docs/source/dataset.rst

```diff
@@ -0,0 +1,44 @@
+===============
+Sample Dataset
```

Comment:
Plural here, as there is more than one data set on this page. Also, as already mentioned, the file should be renamed from `dataset.rst` to something less general, for example `sample_datasets.rst` would fit. Otherwise this can easily be confused with the "Loading data" section, when only looking at the file names.